update sql-statement-modify-column.md (#2829)
pingcap/docs: sql-statements/sql-statement-modify-column.md

  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin AUTO_INCREMENT=30001
1 row in set (0.00 sec)
```

## MySQL compatibility

* Making multiple changes in a single `ALTER TABLE` statement is not currently supported. For example:

    ```sql
    ALTER TABLE t1 MODIFY col1 BIGINT, MODIFY id BIGINT NOT NULL;
    ERROR 1105 (HY000): Unsupported multi schema change
    ```

> * Lossy data type changes and some of the other type changes are not supported(including changes from integer to string or `BLOB`). For example:

Reviewer: ```suggestion
* Does not support changes of lossy data types and some other types (including changes from integer to string or to `BLOB`). For example:
```

    ```sql
    CREATE TABLE t1 (col1 BIGINT);
    ALTER TABLE t1 MODIFY col1 INT;
    ERROR 8200 (HY000): Unsupported modify column length 11 is less than origin 20
    ```

> * Modifying precision of the `DECIMAL` data type is not supported. For example:

Reviewer: ```suggestion
* Does not support modifying the precision of the `DECIMAL` data type. For example:
```
